### PR TITLE
Add test suites for Aptos and fix minor bugs

### PR DESCRIPTION
**Pull request by octave08**
### 📦 Pull Request

Screenshot 2023-06-06 at 9 16 11 AM

updated
Screenshot 2023-06-06 at 10 07 45 AM

[Provide a general summary of the pull request here.]

Add AptosExtension test suite

- scaffold test environment with `yarn scaffold`

### ✅ Fixed Issues

- [List any fixed issues here like: Fixes #XXXX]

Add aptos-extension.spec.ts

### 🚨 Test instructions

[Describe any additional context required to test the PR/feature/bug fix.]

### ⚠️ Don't forget to add a [semver](https://semver.org/) label! 
Please only add **one** label:
- `patch`: Bug Fix?
- `minor`: New Feature?
- `major`: Breaking Change?
- `skip-release`: It's unnecessary to publish this change.

### CODE REVIEW

**Review by karljin — APPROVED** [review]

File: packages/@magic-ext/aptos/src/index.ts [hunk 1]
```diff
@@ -96,7 +98,7 @@ export class AptosExtension extends Extension.Internal<'aptos', any> {
     const transactionBytes = this.serializeRawTransaction(rawTransaction);
 
     return this.request<{ hash: Types.HexEncodedBytes }>(
-      this.utils.createJsonRpcRequestPayload(AptosPayloadMethod.AptosSignAndSubmitTransaction, [
+      this.utils.createJsonRpcRequestPayload(AptosPayloadMethod.AptosSignAndSubmitBCSTransaction, [
```
Comment (octave08) [inline packages/@magic-ext/aptos/src/index.ts:101]: Typo bug

File: packages/@magic-ext/aptos/src/MagicAptosWallet.ts [hunk 2]
```diff
@@ -158,11 +158,13 @@ export class MagicAptosWallet implements AdapterPlugin {
 
   async onNetworkChange(callback: any): Promise<void> {
     console.warn('onNetworkChange is not supported');
+    callback();
```
Comment (octave08) [inline packages/@magic-ext/aptos/src/MagicAptosWallet.ts:161]: added for testing

File: packages/@magic-ext/aptos/src/MagicAptosWallet.ts [hunk 3]
```diff
@@ -158,11 +158,13 @@ export class MagicAptosWallet implements AdapterPlugin {
...
     return Promise.resolve();
   }
 
   async onAccountChange(callback: any): Promise<void> {
     console.warn('onAccountChange is not supported');
+    callback();
```
Comment (octave08) [inline packages/@magic-ext/aptos/src/MagicAptosWallet.ts:167]: added for testing

File: packages/@magic-ext/aptos/src/MagicAptosWallet.ts [hunk 4]
```diff
@@ -38,16 +38,16 @@ export class MagicAptosWallet implements AdapterPlugin {
...
     const isLoggedIn = await this.provider.user.isLoggedIn();
     if (isLoggedIn) {
       const accountInfo = await this.account();
       return accountInfo;
     }
 
+    if (!this.config) {
+      throw new Error('Please set connect config first');
+    }
+
```
Comment (octave08) [inline packages/@magic-ext/aptos/src/MagicAptosWallet.ts:50]: Bug: config should be checked when used

**Review by karljin — APPROVED** [review]

**Comment by magiclabsFE** [comment]
:rocket: PR was released in [`@magic-ext/aptos@0.5.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/aptos%400.5.0) :rocket:

**Pull request merged**